packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -144,6 +144,17 @@ const propTypes = {
 
   /**
    * @private
+   * The column span value for a column.
+   */
+  columnSpan: PropTypes.number,
+
+  /**
+   * @private
+   * The column span index value for a column.
+   */
+  columnSpanIndex: PropTypes.number,
```

Comment:
Shouldn't the cell only need access to the columnSpanIndex?  The Cell component should not care about the column span.  It is not a Cell component concern.

Reply:
Updated.